bitwarden_license/src/Portal/Models/PoliciesModel.cs

```diff
@@ -24,6 +24,12 @@ public PoliciesModel(ICollection<Policy> policies, bool useSso)
                 {
                     continue;
                 }
+
+                if (type == PolicyType.ResetPassword && !useResetPassword)
```

Comment:
We might want to consider just passing the full `SelectedOrganizationDetails` object in here instead of pulling out individual properties for parameters. Otherwise this parameter list is just going to balloon with time.

Reply:
Fair. Will update.

Reply:
Updated.